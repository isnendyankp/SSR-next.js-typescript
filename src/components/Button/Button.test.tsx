import Button from ".";
import { render, screen } from "@testing-library/react";

describe('Button component unit testing', () => {
  it('should render button with label', () => {
    render(<Button label="Click me" />);
    const buttonElement = screen.getByText(/click me/i);
    expect(buttonElement).toMatchSnapshot();
  });
});