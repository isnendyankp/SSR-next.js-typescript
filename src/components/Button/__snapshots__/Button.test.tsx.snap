// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component unit testing should render button with label 1`] = `
<button
  class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 border border-blue-700 rounded"
>
  Click me
</button>
`;
